From 690344aab49bc6952020e4f79042878972a61d71 Mon Sep 17 00:00:00 2001
From: Tomas Winkler <tomas.winkler@intel.com>
Date: Wed, 20 Feb 2019 17:56:04 +0200
Subject: [PATCH 1327/1552] mei: hbm: vtag: clean the feature flags on link
 reset

The list of supported functions can be altered upon link reset,
clean the flags to allow correct selections of supported
features.

Change-Id: Ia99f524f11fce587b142795de59e6cb8da913fc9
Tracked-On: PKT-1757
Signed-off-by: Alexander Usyskin <alexander.usyskin@intel.com>
Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
---
 drivers/misc/mei/hbm.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/misc/mei/hbm.c b/drivers/misc/mei/hbm.c
index bf3e530..f996724 100644
--- a/drivers/misc/mei/hbm.c
+++ b/drivers/misc/mei/hbm.c
@@ -1095,12 +1095,16 @@ static void mei_hbm_config_features(struct mei_device *dev)
 		dev->hbm_f_dr_supported = 1;
 
 	/* VM Tag Support */
+
+	dev->hbm_f_vm_supported = 0;
 	if (dev->version.major_version > HBM_MAJOR_VERSION_VM ||
 	    (dev->version.major_version == HBM_MAJOR_VERSION_VM &&
 	     dev->version.minor_version >= HBM_MINOR_VERSION_VM))
 		dev->hbm_f_vm_supported = 1;
 
 	/* Capability message Support */
+
+	dev->hbm_f_cap_supported = 0;
 	if (dev->version.major_version > HBM_MAJOR_VERSION_CAP ||
 	    (dev->version.major_version == HBM_MAJOR_VERSION_CAP &&
 	     dev->version.minor_version >= HBM_MINOR_VERSION_CAP))
-- 
2.7.4

